pkg/lwip/contrib/sock/lwip_sock.c

```diff
@@ -497,12 +523,20 @@ ssize_t lwip_sock_send(struct netconn **conn, const void *data, size_t len,
     if (remote != NULL) {
         err = netconn_sendto(tmp, buf, &remote_addr, remote_port);
     }
+#if LWIP_TCP
+    else if (tmp->type & NETCONN_TCP) {
+        err = netconn_write_partly(tmp, data, len, 0, &written);
```

Comment:
can't you reuse `res` instead of introducing an addition var with `written`? As far as I see, you set `res = written` anyway afterwards?!